job_runner/reporting/interactive.py

```diff
@@ -1,20 +1,27 @@
-from job_runner.reporting.message import Finish, Start
+import logging
+
+from job_runner.reporting.message import Message, Finish, Start
 from job_runner.reporting.base import Reporter
 
+logger = logging.getLogger(__name__)
+
 
 class Interactive(Reporter):
-    def report(self, msg):
+    def _report(self, msg: Message) -> str:  # type: ignore
         if isinstance(msg, Start):
-            print("Running job: {} ... ".format(msg.job.name()))
-        elif not msg.success() and not isinstance(msg, Finish):
-            print(
-                """failed ....
+            return f"Running job: {msg.job.name()} ... "
+        if isinstance(msg, Finish):
+            return (
+                "OK"
+                if msg.success()
+                else f"""failed ....
             -----------------------------------------------------------------
-            Error: {}
+            Error: {msg.error_message}
             -----------------------------------------------------------------
-            """.format(
-                    msg.error_message
-                )
+            """
             )
-        elif isinstance(msg, Finish) and msg.success():
-            print("OK")
+
+    def report(self, msg: Message):
+        msg_ = self._report(msg)
+        print(msg_)
+        logger.debug(msg_)
```

Comment:
This would work, but it relies on another reporter. I would suggest either writing a (very simple) custom reporter, or (I think) even better, add a `logger.info(msg)` here: https://github.com/equinor/ert/blob/8130abf0ae50dc4376c44429a04d22c5dba46609/job_runner/cli.py#L73

Then we leave it up to the logger to filter out data, and we dont impose restrictions in `ert`.

Reply:
Not sure what is that needs to be done here. But I did the minimum you asked for and switch to info.

Reply:
Now the logging statement is inside the `Interactive` reporter, I would say it makes more sense to have it on line 74 of `ert/job_runner/cli.py` above, because then it is not tied to a specific reporter (as they serve a different purpose)